### ISSUE TITLE
level and mod variable must not be changeable while in game

### ISSUE THREAD

**Issue by albertz** [openlierox/openlierox#309]
Converted from [SourceForge issue 2804744](http://sourceforge.net/support/tracker.php?aid=2804744), submitted by albertzeyer

Btw., the dedicated script is also a bit buggy when it is restarted  
and you are in a game. It changes the level/mod variable but for the  
current game, for the local server client, the old level/mod of course  
are still active and all new connecting people will get the new level/ 
mod. That gives some strange behaviour. (Though no crash and it's  
fixed when you return to lobby.) To fix that, the setVar command  
should perhaps check that some of the variables can not be changed  
when the game is running (at least mod/level; most others can be  
changed also in game).

I have already a similar fix for the password variable in getVar and in chatcommand setVar.

Basically, both setVar of chat command and of real commands must have another check if it is one of these variables while ingame.

**Comment by albertz** [comment]
Submitted by albertzeyer

Fixed in 4694.
